src/OpenLoco/Message.h

```diff
@@ -58,8 +58,8 @@ namespace OpenLoco
     {
         constexpr uint16_t unk0 = 1 << 0;
         constexpr uint16_t unk1 = 1 << 1;
-        constexpr uint16_t unk2 = 1 << 2;
-        constexpr uint16_t unk3 = 1 << 3;
+        constexpr uint16_t hasFirstItem = 1 << 2;
+        constexpr uint16_t hasSecondItem = 1 << 3;
```

Comment:
Would the use of `Entity` be appropriate here?

Reply:
Hmm not really its used to indicate if it has any argument so could be entity, station, location, town, ...